backend/models/Transaction.js

```diff
@@ -40,7 +40,7 @@ module.exports = (sequelize, DataTypes) => {
     {
       tableName: "transactions",
       underscored: true,
-      timestamps: false,
+      timestamps: true,
```

Comment:
This is not useful as synchronization queries nearcore in "arbitrary" order, and thus the timestamps added on the DB level are just useless. Let's discuss what you want to achieve, so I can help you.

Reply:
we need the time when new account created.